cli/src/argparse/subcommand.rs

```diff
@@ -424,6 +433,66 @@ impl Sync {
     }
 }
 
+struct Import;
+
+impl Import {
+    fn parse(input: ArgList) -> IResult<ArgList, Subcommand> {
+        fn to_subcommand(_: &str) -> Result<Subcommand, ()> {
+            Ok(Subcommand::Import)
+        }
+        map_res(arg_matching(literal("import")), to_subcommand)(input)
+    }
+
+    fn get_usage(u: &mut usage::Usage) {
+        u.subcommands.push(usage::Subcommand {
+            name: "import",
+            syntax: "import",
+            summary: "Import tasks",
+            description: "
+                Import tasks into this replica.
+
+                The tasks must be provided in the TaskWarrior JSON format on stdin.  If tasks
+                in the import already exist, they are 'merged'.
+
+                Because TaskChampion lacks the information about the types of UDAs that is stored
```

Comment:
It feels a bit odd to have the `ta import` command be TW specific (rather than be a TC-specific "roundtripable" format), but, given the direction of things this is probably fine :shrug:

Reply:
Maybe makes sense to have `ta import` be name `ta import-tw` or similar? Also ":shrug:"